1.5/Source/PawnStorages/PawnStorages/Utility.cs

```diff
@@ -124,4 +135,35 @@ public static Job ReleaseJob(CompPawnStorage store, Pawn releaser, Pawn toReleas
 
         return JobMaker.MakeJob(PS_DefOf.PS_Release, store.Parent, null, toRelease);
     }
-}
+
+    public static List<ThingDef> animals;
+
+    public static bool CompPropertiesIsProducer(CompProperties c)
+    {
+        return c.compClass == typeof(CompEggLayer) || c.compClass.IsSubclassOf(typeof(CompEggLayer)) ||
```

Comment:
Can be simplified with IsAssignableFrom. Something like
```
typeof(CompEggLayer).IsAssignableFrom(c.compClass) || typeof(CompHasGatherableBodyResource).IsAssignableFrom(c.compClass);
```

Reply:
It keeps bloody deleting `PawnStorages.VEF.dll` for me. Damned file refuses to exist on my system.